src/game/Dice.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { css, StyleSheet } from 'aphrodite';
+
+import { ReactComponent as DiceOne } from '../static/dice/dice-one.svg';
+import { ReactComponent as DiceTwo } from '../static/dice/dice-two.svg';
+import { ReactComponent as DiceThree } from '../static/dice/dice-three.svg';
+import { ReactComponent as DiceFour } from '../static/dice/dice-four.svg';
+import { ReactComponent as DiceFive } from '../static/dice/dice-five.svg';
+import { ReactComponent as DiceSix } from '../static/dice/dice-six.svg';
+import { ReactComponent as DiceHidden } from '../static/dice/dice-hidden.svg';
+
+type Props = Readonly<{
+  value: number;
+  isHidden: boolean;
+  ownership: 'self' | 'other';
+}>;
+
+type DiceProps = Readonly<{

```

Comment:
Same as above. Possibly rename it to diceProps? I looked through some of the other code and there was some inconsistencies with naming. For example we have JoinLobby, and onJoinLobby both as function names. That is another topic all together